src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="Azure.Data.AppConfiguration" Version="1.3.0-beta.2" />
     <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.3.0" />
+    <PackageReference Include="DnsClient" Version="1.7.0" />
```

Comment:
What? MS package depending on Open Source? Everyone who references App Configuration now needs to take on an open source dependency? Was this discussed?

Reply:
Yes, just added you to the discussion thread.

Reply:
According to the offline discussion, we move forward to use this OSS project , just resolve this